app/src/main/java/com/rob729/quiethours/Fragments/NewProfileFragment.kt

```diff
@@ -177,18 +166,27 @@ class NewProfileFragment : Fragment() {
                 binding.noteTextInput.visibility = VISIBLE
                 binding.noteCheckBox.isChecked = true
             }
-            binding.StartTime.setText("${Utils.setTimeString(args.shr)}:${Utils.setTimeString(args.smin)}")
+            binding.StartTime.setText(setString(args.shr, args.smin))
             shr = args.shr
             smin = args.smin
             ehr = args.ehr
             emin = args.emin
-            binding.EndTime.setText("${Utils.setTimeString(args.ehr)}:${Utils.setTimeString(args.emin)}")
+            binding.EndTime.setText(setString(args.ehr, args.emin))
             binding.vibSwitch.isChecked = args.vibSwitch
             binding.repeatWeeklySwitch.isChecked = args.repeatWeekly
             binding.makeProfileFab.text = "UPDATE"
         }
         return binding.root
     }
+    private fun setStringFormat(editText: EditText) {
```

Comment:
setStringFormat() and setString() are used to do the same thing. 
Have a closer look and refactor the code.